both/methods.js

```diff
@@ -176,8 +185,9 @@ Meteor.methods({
     assert(round); // canRemoveRound checked that roundId is valid
 
     Rounds.remove({ _id: roundId });
-    RoundProgresses.remove({ roundId: roundId });
-    Groups.remove({ roundId: roundId });
+    [RoundProgresses, Results, Groups, ScheduleEvents].forEach(function(collection) {
```

Comment:
Note to Jeremy: How do we delete `ScheduleEvents` that don't have a corresponding `Round`?